data/hullmods/hull_mods.csv

```diff
@@ -1,38 +1,40 @@
-,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,
 Oversized Hull,diableavionics_oversized,,,,,,,,TRUE,,0,0,0,0,data.hullmods.DiableAvionicsOversized,This ship is abnormally large which creates a logistical nightmare when it needs to be readied for combat. Increases CR recovery cost by %s.,Increases CR recovery cost by %s.,,graphics/da/hullmods/diableavionics_oversized.png
-,,,,,,,,,,,,,,,,,,
-,,,,,,,,,,,,,,,,,,
-Dampened Mount,diableavionics_mountBI,4,,,,,,,TRUE,,0,0,0,0,data.hullmods.DiableAvionicsMountBI,The hull's weapon emplacements are designed around extra recoil dampeners allow projectile weapons to fire at a slightly extended range of %s.,,,graphics/da/hullmods/diableavionics_mount.png
-#Dampened Mount,diableavionics_mount,3,0.33,Diable Avionics,"offensive, req_spaceport","Weapons, Requires Dock",15000,,,,10,15,25,40,data.hullmods.DiableAvionicsMount,Extra recoil dampeners are fitted around the mounts allowing projectile weapons to fire at a slightly extended range of %s but only at %s of their maximum firerate. They also take %s more damage as a result.,,,graphics/da/hullmods/diableavionics_mount.png
-#Dampened Mount,diableavionics_mount,4,,,,,,,TRUE,,0,0,0,0,data.hullmods.DiableAvionicsMount,"This hullmod should no longer exist",,,graphics/da/hullmods/diableavionics_mount.png
-,,,,,,,,,,,,,,,,,,
-# WANZER GANTRY
-,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,
+Dampened Mount,diableavionics_mountBI,4,,,,,,,TRUE,,0,0,0,0,data.hullmods.DiableAvionicsMountBI,"The hull's weapon emplacements are designed around extra recoil dampeners allow projectile weapons to fire at a slightly extended range of %s and %s recoil decreased.And if the projectile weapons are in large slots，gain %s 
+instead.",,"The ship`s weapon slots were redesigned and improved to integerate recoil dampers in the ship and reduced the op of installing diable avionics ballistc and energy weapons by %s/%s/%s according to size.It also counteract the penalty of fire rate if it exists.
+Hovever such a complex modification raise the ship`s dp by %s/%s/%s/%s according to ship size.",graphics/da/hullmods/diableavionics_mount.png
+Dampened Mount,diableavionics_mount,3,0.33,Diable Avionics,"offensive, req_spaceport","Weapons, Requires Dock",15000,,,,8,12,20,30,data.hullmods.DiableAvionicsMount,Extra recoil dampeners are fitted around the mounts allowing projectile weapons to fire at a slightly extended range of %s and %s recoil decreased but only at %s of their maximum firerate. They also take %s more damage as a result.,,"The ship`s weapon slots were redesigned and improved to integerate recoil dampers in the ship and reduced the op of installing diable avionics ballistc and energy weapons by %s/%s/%s according to size.It also counteract the penalty of fire rate if it exists.
+Hovever such a complex modification raise the ship`s dp by %s/%s/%s/%s according to ship size.",graphics/da/hullmods/diableavionics_mount.png
```

Comment:
Dampened Mount is modular and can be installed on Diable Avionics ships now. If installed on a ship that doesn't have it built-in, the firerate of the ship is reduced and the weapons also take more damage.

Either version of the hullmod can be s-modded to reduce the OP cost of Diable Avionics weapons while increasing the DP of the ship.